apps/publikator/components/editor/modules/link/ui.js

```diff
@@ -222,6 +223,14 @@ export const LinkForm = withT(
                 value={node.data.get('title')}
                 onChange={onInputChange('title')}
               />
+              <Field
+                label={t(`link/description`, undefined, 'title')}
+                value={node.data.get('description')}
+                onChange={onInputChange('description')}
+                renderInput={({ ref, ...inputProps }) => (
+                  <AutosizeInput {...inputProps} inputRef={ref} />
```

Comment:
<img width="295" alt="Screenshot 2022-07-21 at 01 24 30" src="https://user-images.githubusercontent.com/410211/180099193-7bcdb362-fde6-417a-9e78-60d6b4b59892.png">

needs `{...styles.autoSize}` to look properly (min height and paddings)

Reply:
thanks for the tip!